docs/DeveloperGuide.adoc

```diff
@@ -1250,16 +1246,26 @@ These instructions only provide a starting point for testers to work on; testers
 
 _{ more test cases ... }_
 
+=== Adding a loan
+
+. Adding a loan while all loans are listed
+
+.. Prerequisites: List all loans using the `list` command. Multiple loans in the list.
+.. Test case: `PLACEHOLDER` +
+   Expected: PLACEHOLDER
+
 === Deleting a loan
 
 . Deleting a loan while all loans are listed
 
 .. Prerequisites: List all loans using the `list` command. Multiple loans in the list.
-.. Test case: `delete 1` +
+.. Test case: `delete i/1 x/a12345` +
    Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-.. Test case: `delete 0` +
+.. Test case: `delete i/1 x/a12345` +
+   Expected: No loan is deleted. Error details shown in the status message. Status bar remains the same.
+.. Test case: `delete i/1 x/wr0ngP4sS` +
```

Comment:
Can add a note at the end "(wrong password)" to make it clear what is wrong. For all I know, the user really did set the password to `wr0ngP4sS`.